app/views/layouts/_call_list_messages.html.erb

```diff
@@ -0,0 +1,6 @@
+<div id='flash'>
```

Comment:
I think this might be redundant to `app/views/layouts/_messages` -- can we use regular `_messages` instead?